src/components/EditCoursePage/EditCourseForm.test.jsx

```diff
@@ -32,6 +32,8 @@ describe('BaseEditCourseForm', () => {
     uuid: '11111111-1111-1111-1111-111111111111',
     editable: true,
     skill_names: [],
+    organization_logo_override: 'http://image.src.small',
+    organization_short_code_override: 'test short code',
```

Comment:
where are they verified?

Reply:
It will be verified in Snapshot
https://github.com/openedx/frontend-app-publisher/pull/731/files#diff-92f8bc89159dd71818dc91467b10a47115d4966f2705b5ac3138bf64b54f95a7R948